shell/conf/benchmarks_defaults.conf

```diff
@@ -26,10 +26,14 @@ if [ "$BENCH_SCALE_FACTOR" ] ; then
 else
    if [ ! "$BENCH_DATA_SIZE" ] ; then
      # Check if TPC scale factor has been specified
-     if [ "$TPCH_SCALE_FACTOR" ] ; then
-       BENCH_DATA_SIZE="$(( TPCH_SCALE_FACTOR *  1000000000 ))" # if set, it is in GBs
+     if [[ "$TPCH_USE_LOCAL_FACTOR" > 0 ]]; then 
+       BENCH_DATA_SIZE="$(( TPCH_USE_LOCAL_FACTOR *  1000000000 ))" # if set, it is in GBs
      else
-       BENCH_DATA_SIZE="100000000" #100MB, in bytes #1000000000000 1TB
+       if [ "$TPCH_SCALE_FACTOR" ] ; then
+         BENCH_DATA_SIZE="$(( TPCH_SCALE_FACTOR *  1000000000 ))" # if set, it is in GBs
+       else
+         BENCH_DATA_SIZE="100000000" #100MB, in bytes #1000000000000 1TB
+       fi
```

Comment:
Very similar code to `common_TPC-H.sh` - can we avoid this?

Reply:
Yeah, providing a solution in an upcoming branch